src/pages/NoteBooks.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Item from "../components/ItemList/ItemNotebook";
+import s from "./NoteBooks"
```

Comment:
You sould import styles from "./Notebools.module.css" 
./Notebooks is React Component but isn't a file with styles